test/dom-snapshots/elements-hx-progress-chrome.js.md

```diff
@@ -0,0 +1,117 @@
+# Snapshot report for `built/dom-snapshots/elements-hx-progress-chrome.js`
+
+The actual snapshot is saved in `elements-hx-progress-chrome.js.snap`.
+
+Generated by [AVA](https://ava.li).
+
+## chrome on macOS 10.13 -- elements/hx-progress/test.html
```

Comment:
If you're running these tests locally on saucelabs, you should stop. The travis-ci build server will run the browser locally using a local selenium server....which means the initial checked in version must do the same.